src/Components/Cards/AvailableModelCard/AvailableModelCard.js

```diff
@@ -0,0 +1,104 @@
+//#region Import Neccessary Dependencies
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+//#endregion
+
+// Import Stylings
+import './AvailableModelCard.css';
+
+//#region Import UI Components
+import IconButton from '../../Buttons/IconButton/IconButton';
+//#endregion
+
+//#region Import Icons
+import { faScrewdriverWrench } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { MdCheckBox, MdCheckBoxOutlineBlank } from 'react-icons/md';
+//#endregion
+
+// Define AvailableModelCard Component
+function AvailableModelCard(props) {
+
+  // Destructure props
+  const { className, modelId, modelName, modelPhoto, typeName, typeId, availableCount, isSelected, onSelect, isMakingReservation } = props;
+
+  // State variable to manage model photo
+  const [equipmentModelPhoto, setEquipmentModelPhoto] = useState(modelPhoto);
+
+  // Text for item count
+  const itemText = availableCount > 1 ? 'items' : 'item';
+
+  // HandleOnSelect - Handle model selection
+  const HandleOnSelect = () => {
+    onSelect({
+      modelId,
+      modelName,
+      modelPhoto,
+      typeName,
+      typeId,
+      availableCount,
+      quantity: 1,
+    });
+  };
+
+  return (
+    <div className={`AvailableModelCard-Container ${isSelected ? 'AvailableModelCard-Active' : ''} ${className}`}>
+      <button className='AvailableModelCard-ModelPhoto'
+        onClick={isMakingReservation ? HandleOnSelect : undefined}>
+          {/* Render model photo or default icon if photo not available */}
+          {equipmentModelPhoto && (
+            <img src={modelPhoto}
```

Comment:
Sonarlint's giving me this:
![interactive-error](https://github.com/tranleduyan/ETC-Frontend/assets/97055517/1e3102c5-c77b-4c2b-a43d-52b80851a13c)
![interactive-error-sol](https://github.com/tranleduyan/ETC-Frontend/assets/97055517/0f7c18ed-9620-42b8-a97b-8f65f549d0b2)
I didn't look too much into this, so you can take it or leave it 😉

Reply:
Oh year, it was because of the OnError function of img and img does not seen as button. However, there seems to not having the other ways to treat it as button so I think we will just go with it for now!